src/modules/UI/scenes/OnBoarding/OnBoardingScene.js

```diff
@@ -0,0 +1,111 @@
+// @flow
+import React, { Component } from 'react'
+import { Animated, Modal, View } from 'react-native'
+
+import PagingDots from '../../../../connectors/components/OnboardingPagingDotsConnector.js'
+import s from '../../../../locales/strings.js'
+import { OnBoardingSceneStyles } from '../../../../styles/indexStyles.js'
+import { PLATFORM } from '../../../../theme/variables/platform'
+import { PrimaryButton } from '../../components/Buttons/'
+import OnBoardingSlide from './OnBoardingSlide'
+
+// import Swiper from 'react-native-swiper'
+
+type Props= {
```

Comment:
Should be `OnBoardingOwnProps` and then `OnBoardingState`

Reply:
This should have failed eslint since there is no space after `=`. Was a linter skipped on precommit?